app/modules/transcribe/overlay/transcribe-dialog.directive.js

```diff
@@ -116,8 +116,13 @@ function transcribeDialogController($rootScope, $element, $timeout, AnnotationsF
 
     function saveAndCloseDialog() {
         if (vm.transcription !== vm.data.text) {
-            vm.data.text = vm.transcription;
-            AnnotationsFactory.upsert(vm.data);
+            if (vm.data.type === 'marginalia') {
+                vm.data.text = '<label>' + vm.transcription + '</label>';
+                AnnotationsFactory.upsert(vm.data);
+            } else {
+                vm.data.text = vm.transcription;
+                AnnotationsFactory.upsert(vm.data);
```

Comment:
You could move this line after the if/else block, and delete 121 as they're doing the same thing